meson.build

```diff
@@ -5,6 +5,14 @@ project('libratbag', 'c', 'cpp',
 
 libratbag_version = meson.project_version().split('.')
 
+ratbagd_dir = join_paths(meson.current_source_dir(), 'ratbagd')
+r = run_command('/usr/bin/git', 'log --format=format:%h -1 -- @0@'.format(ratbagd_dir).split())
```

Comment:
two things here: why specify the full path to git? meson approach here would be (I think):

```
git = find_program('git')
run_command(git, ....)
````

The other thing is that the meson doc say that this is run from an unspecified directory, you can't assume that we're in the source directory here. This is something we have to move to a script, using `GIT_DIR=$MESON_SOURCE_ROOT/.git` or some pushd/popd combination.